mgmtsystem/google.py

```diff
@@ -127,9 +128,17 @@ def list_image(self):
 
     def _find_instance_by_name(self, instance_name):
         try:
-            instance = self._instances.get(
-                project=self._project, zone=self._zone, instance=instance_name).execute()
-            return instance
+            # by default search the instances in current zone..
+            for instance in self._get_zone_instances(self._zone).get('items', []):
```

Comment:
@psav kept the first call as it is (original version), if exception in first call then searching the instance in all other zones. raise the VMInstanceNotFound exception(original version) if, even the instance not found with the second call.